GatherPress/gatherpress
# Adding user formats and timezone

### Description of the Change
Allow subscribers to define their own date/time format and timezone

Steps to verify:

1. Log in as an end user, and go to the edit profile page.
2. You should see **Date & Time Formatting** with some fields under it.
3. Add some custom formats ( you should see the preview functionality working).
4. Go to the event page, and you should see the date/time formats match what you entered.




Closes #446

### How to test the Change
Testing instructions are above with the verification.



### Changelog Entry

> Added - New feature
Allow subscribers to define their own date/time format and timezone

### Credits

Props @linusx

### Checklist:
